src/core/public/application/utils/get_app_info.test.ts

```diff
@@ -59,25 +55,23 @@ describe('getAppInfo', () => {
       status: AppStatus.accessible,
       navLinkStatus: AppNavLinkStatus.visible,
       appRoute: `/app/some-id`,
-      meta: {
-        keywords: [],
-        searchDeepLinks: [
-          {
-            id: 'sub-id',
-            title: 'sub-title',
-            keywords: [],
-            searchDeepLinks: [
-              {
-                id: 'sub-sub-id',
-                title: 'sub-sub-title',
-                path: '/sub-sub',
-                keywords: [],
-                searchDeepLinks: [], // default empty array added
-              },
-            ],
-          },
-        ],
-      },
+      keywords: [],
+      deepLinks: [
+        {
+          id: 'sub-id',
+          title: 'sub-title',
+          keywords: [],
+          deepLinks: [
```

Comment:
Shouldn't the top-level deeplinks have a default `navLinkStatus`? We may need a separate type for the nested deep links since those should not have `navLinkStatus`.